starter.flow.better/src/main/java/com/example/starter/flow/better/osgi/VaadinMode.java

```diff
@@ -0,0 +1,40 @@
+package com.example.starter.flow.better.osgi;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.osgi.service.component.annotations.ComponentPropertyType;
+import org.osgi.service.http.whiteboard.annotations.RequireHttpWhiteboard;
+
+/*
+ * @author Stefan Bischof
+ */
+
+@ComponentPropertyType
+@Retention(RetentionPolicy.CLASS)
+@Target(ElementType.TYPE)
+@RequireHttpWhiteboard
+public @interface VaadinMode {
+
+	String PREFIX_ = "servlet.init.";
```

Comment:
The constant is not used from anywhere.

Is it some magic ?
Is it documented somewhere ?